quoridor-app/package.json

```diff
@@ -20,6 +21,7 @@
     "react-navigation": "3.11.0"
   },
   "devDependencies": {
+    "@testing-library/react-native": "^4.0.7",
```

Comment:
It is better to stay on an exact version, i.e. to remain consistent with other dependencies. With npm, you can use the `--save-exact` option.